scripts/trlite

```diff
@@ -298,7 +298,7 @@ function check_jstring_util() {
     CALLCOUNT=$(grep jerry_string_to_char_buffer src/zjs_util.c | wc -l)
 
     # make sure there are exactly two calls to the function
-    if [ "$CALLCOUNT" != "2" ]; then
+    if [ $CALLCOUNT != "2" ]; then
```

Comment:
Are you sure this is needed? Usually the quotes are sort of optional but kind of safer in case it ever got a result with a space in it, so BKM AFAIK.